src/libraries/System.Text.Json/src/System/Text/Json/Serialization/JsonNamingPolicy.cs

```diff
@@ -26,6 +28,7 @@ public abstract class JsonNamingPolicy
         /// </summary>
         /// <param name="name">The name to convert.</param>
         /// <returns>The converted name.</returns>
-        public abstract string ConvertName(string name);
+        [return: NotNullIfNotNull("name")]
+        public abstract string? ConvertName(string? name);
```

Comment:
This method should not return null strings (nor will the JsonSerializer ever pass `null` as an argument.

```suggestion
        public abstract string ConvertName(string name);
```

Any places where we are not guarding against a null string returned from `ConvertName`, we should. For example:
https://github.com/dotnet/runtime/blob/583afbb88879cd749866f67338b47ee1bc9c9f63/src/libraries/System.Text.Json/src/System/Text/Json/Serialization/JsonSerializer.Write.HandleDictionary.cs#L86

Reply:
OK will update